lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Patron.java

```diff
@@ -59,6 +59,18 @@ public Set<Book> getCheckedOutBooks() throws LibraryNotSetException {
     return this.library.getCheckedOutByPatron(this);
   }
 
+<<<<<<< HEAD
```

Comment:
Looks like you had a bad merge. I think you can merge using VS Code with the 3-way merge editor:

https://code.visualstudio.com/docs/sourcecontrol/overview#_3way-merge-editor